packages/kernel-6.1/kernel-6.1.spec

```diff
@@ -187,10 +220,18 @@ make -s\\\
 %kmake %{?_smp_mflags} %{_cross_kimage}
 %kmake %{?_smp_mflags} modules
 
+%if "%{_cross_arch}" == "x86_64"
+%kmake %{?_smp_mflags} M=%{_builddir}/neuron
+%endif
+
 %install
 %kmake %{?_smp_mflags} headers_install
 %kmake %{?_smp_mflags} modules_install
 
+%if "%{_cross_arch}" == "x86_64"
```

Comment:
Should we track somewhere when Neuron supports ARM?

Reply:
> Should we track somewhere when Neuron supports ARM?

I don't think so. If EC2 ever launches an instance type that combines Neuron and Graviton, we can remove the conditional.